ts/a11y/explorer.ts

```diff
@@ -260,11 +265,10 @@ export function ExplorerMathDocumentMixin<B extends MathDocumentConstructor<HTML
         infoRole: false,                   // show semantic role on mouse hovering
         infoType: false,                   // show semantic type on mouse hovering
         keyMagnifier: false,               // switch on magnification via key exploration
-        locale: 'en',                      // switch the locale
```

Comment:
Something similar to the above, but setting `sre.locale` would be good, too.

Reply:
`locale` is not explicitly set in `sre` whenever `a11y` value changes. I.e., `a11y` value dominates `sre` value.